### PR TITLE
TaskRow: Make task descriptions small

### PR DESCRIPTION
**Pull request by danrabbit**
Alternative to #52

### CODE REVIEW

**Comment by cassidyjames** [comment]
@danrabbit does it make more sense to use CSS or pango `<small>` here?

**Comment by danrabbit** [comment]
@cassidyjames Yeah can do. I just wanted to make sure to match between the description text and due label. Maybe I should change them both to Pango?

I'm wondering about if we should have a CSS class for "SMALL_LABEL". Using Pango seems to make things more complicated when the labels needs to change

**Comment by cassidyjames** [comment]
@danrabbit yeah a constant in Granite could make sense, and matches the convention with dim label. We should probably make it match whatever Pango does to start for consistency, but once we move things over to it, we'd have more freedom with the styling.

File: src/Widgets/TaskRow.vala [hunk 1]
```diff
@@ -86,7 +86,12 @@ public class Tasks.TaskRow : Gtk.ListBoxRow {
         description_label.xalign = 0;
         description_label.lines = 1;
         description_label.ellipsize = Pango.EllipsizeMode.END;
-        description_label.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
+        description_label.use_markup = true;
+
+        unowned Gtk.StyleContext description_label_context = description_label.get_style_context ();
+        description_label_context.add_provider (taskrow_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
+        description_label_context.add_class ("description");
```
Comment (cassidyjames) [inline src/Widgets/TaskRow.vala]:
```suggestion
        // NOTE: This should change to using the constant after a release of Granite with elementary/granite#367
        description_label_context.add_class ("small-label");
```

**Comment by jeremypw** [comment]
@danrabbit This was approved a while back but now has several conflicts - is it still relevant?

**Comment by danrabbit** [comment]
Yeah I'm just gonna close this and we can revisit it at another time There are other styling things that need to be addressed before initial release

**Pull request closed without merging**